.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
```

Comment:
Why are we adding this with this PR? Is this required for myaccount? I'd prefer to have a separate conversation about pre-commit hooks and handle it in a separate PR. Also why npx instead of yarn?

Reply:
I feel we will want to take doc/code consistency as the first priority. In this PR, docs for myaccount is generated with code/types, and it easy to miss running doc update script when code changes happen, and the pre-commit here is a solution to keep things in sync. 

I understand it may affect a bit devex, but here I would prefer to take consistency over devex.

Reply:
Could this be done as a pre-push step instead of pre-commit? When rebasing we do a lot of commits, but there is only one push at the end.

Reply:
pre-push cannot just handle staged files, this lint staged won't run scripts when no file matches the regex.